code/main.cpp

```diff
@@ -1,9 +1,31 @@
-#include "hwlib.hpp"
-
+#include <hwlib.hpp>
```

Comment:
Maybe we should move this file to an examples directory. Otherwise this file will change every time we want to push a different screen